builtin/mainmenu/dlg_create_world.lua

```diff
@@ -91,8 +91,19 @@ local function create_world_buttonhandler(this, fields)
 			local message = nil
 
 			core.setting_set("fixed_map_seed", fields["te_seed"])
-
-			if not menudata.worldlist:uid_exists_raw(worldname) then
+			
+			local game_name = gamemgr.get_game(gameindex).id
+			
```

Comment:
This whole section needs improvement:
```
		if gameindex ~= nil and
			worldname ~= "" then

			local message = nil

			core.setting_set("fixed_map_seed", fields["te_seed"])
			
			local game_name = gamemgr.get_game(gameindex).id
			
			local exists = false
```
The 'if' line doesn't need splitting and all following empty spaces can be removed.